applications/GeoMechanicsApplication/tests/shell_with_reset_displacement/README.md

```diff
@@ -0,0 +1,24 @@
+# Shell with reset displacement
+
+This tests verifies that in a shell, the prescribed displacement is not erased by the 'reset_displacement' functionality
+when that option is switched on.
+
+## Setup
+
+This test consists of a single 4-noded element (using the ShellThinElementCorotational3D4N class) of 2000x2000 meter. A
+schematic representation can be found in the figure below:
+![MeshStructure](MeshStructure.svg)
+
+Nodes 1, 2 and 4 have fixed rotations and displacements in all directions. Node 3 has a fixed displacement in the Z
+direction of -20m. The material is described using:
+
+-   A linear elastic plane stress model (LinearElasticPlaneStress2DLaw)
+-   A Young modulus of 4.32e8 with a 0.0 Poisson ratio.
```

Comment:
Young's modulus and Poisson's ratio. ( unless our native Jon speaker thinks otherwise )

Reply:
Wikipedia is okay with both 😄 , but I changed it.